cmd/reading/list/list.go

```diff
@@ -57,23 +69,9 @@ func NewCommand() *cobra.Command {
 			} else {
 				url = config.Conf.Clients["CoreData"].Url() + clients.ApiReadingRoute
 			}
-
-			resp, err := http.Get(url)
-			if err != nil {
-				// handle error
-				fmt.Println("An error occurred. Is EdgeX running?")
-				fmt.Println(err)
-			}
-			defer resp.Body.Close()
-
-			data, _ := ioutil.ReadAll(resp.Body)
-
 			var readings []models.Reading
-			err = json.Unmarshal(data, &readings)
+			err = utils.ListHelper(url, readings)
```

Comment:
should be: err = utils.ListHelper(url, &readings) otherwise the value is not updated and nothing is printed

Reply:
correct. But did we see some compile error yesterday? let me check.

Reply:
Yes, there were some error ..hm.. but we definitely we should send pointer. Now, there is not compilation errors